src/main/java/seedu/address/logic/commands/UpdateCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Updates an Event in the address book.
+ */
+public class UpdateCommand extends Command {
+    public static final String COMMAND_WORD = "update";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates an event in the address book. "
+            + "Parameters: "
+            + PREFIX_INDEX + "EVENT INDEX "
```

Comment:
Might be better to use something more specific like PREFIX_EVENT_INDEX to prevent confusion?